### PR TITLE
create stub file for vp

### PR DESCRIPTION
**Pull request by vp186006**
This is my pull request adding a file...

 @brianamarie @eminbagh @tjroberts

### CODE REVIEW

File: vp186006-kc.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# I am going to add a file 
+# Vasko
+
+## This is git?
+- git is cool
+
+## I'm not in Kansans City
```
Comment (eminbagh) [inline vp186006-kc.md]: @vp186006 Where are you at?

Comment (vp186006) [reply vp186006-kc.md]: @eminbagh LA LA Land...where you should be :+1:

**Comment by tjroberts** [comment]
Should mention some food, like that BBQ pizza...

File: vp186006-kc.md (hunk 1 above)
Comment (eminbagh) [reply vp186006-kc.md]: @vp186006 along with Emma Stone and Ryan Gosling? If so say hi!

**Comment by vp186006** [comment]
@tjroberts  from @eminbagh 's favorite pizza place? :pizza:

**Pull request merged**